test/apiv2/20-containers.at

```diff
@@ -144,6 +144,14 @@ if root; then
     podman rm -f $CTRNAME
 fi
 
+# Verify that compat top endpoint combines multi-entry COMMAND lines
+CTRNAME=testtopproc
+podman run --name $CTRNAME -d $IMAGE sleep 25
+t GET containers/$CTRNAME/top?stream=false 200 \
+  .Processes.[0].[6]="00:00:00" \
+  .Processes.[0].[7]="sleep 25"
+podman rm -f -t0 $CTRNAME
```

Comment:
I guess you could add the output check to the top call above? Any reason to run another container for this?
I mean speed is not that big of a deal in API tests so I can live with that and not block over this.

Reply:
We want a multi-part command - `top` doesn't actually test the combining code as it doesn't have a space in it.

Reply:
ah yes of course